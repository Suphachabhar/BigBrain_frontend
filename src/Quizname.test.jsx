import { shallow } from 'enzyme';
import React from 'react';
import renderer from 'react-test-renderer';
import { Quizname } from './components/Quizname.style';

describe('OptionStyle', () => {
  // test the option style component
  it('uses custom title', () => {
    const title = 'A custom title';
    const quizname = shallow(<Quizname title={title} />);
    expect(quizname.text()).toBe(title);
  });

  it('renders with provided title (snapshot)', () => {
    const quizname = renderer.create(<Quizname title={'title'} />).toJSON;
    expect(quizname).toMatchSnapshot();
  });
});
