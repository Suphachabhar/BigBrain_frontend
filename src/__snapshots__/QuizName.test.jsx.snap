// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OptionStyle renders with provided title (snapshot) 1`] = `[Function]`;
